### ISSUE TITLE
PHP Notice:  unserialize(): Error at offset 0 of 1 bytes in /vqmod/vqmod.php on line 201

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rohitrng/vqmod#76]
```
i solved this problem using by replacing the following line

$this->_mods = unserialize($mods);

with the following lines

if ((bool) trim($mods)) {
    $this->_mods = unserialize($mods);
}
```

Original issue reported on code.google.com by `Abdullah...@gmail.com` on 5 Feb 2013 at 1:57

**Comment by GoogleCodeExporter** [comment]
```
I'm guessing this happened when you had no mods in your /vqmod/xml folder? Will 
add this to the next version (though the (bool) isn't required)
```

Original comment by `DJG6...@gmail.com` on 5 Feb 2013 at 2:00
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Corrected in 2.3.1
```

Original comment by `DJG6...@gmail.com` on 26 Feb 2013 at 1:09
- Changed state: **Fixed**
